### ISSUE TITLE
Author box is inaccessible

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Authors#89]
> The Author management interface on a single post is *super*
> inaccessible. There are more issues than I can list, but the big issues are
> that it's not keyboard accessible and doesn't have accessible labels. The
> "remove author" "x" should be a button element with screen reader text, for
> example. Select2, while undoubtedly very nice for us sighted mouse users,
> is also quite inaccessib

**Comment by htmgarcia** [comment]
@stevejburge using the keyboard is already possible to reach the authors box. Still is a bit inconsistent in comparison with core WP boxes https://youtu.be/Yt3QEODjPIg
The main issue is the arrow to open the box is different to the native ones:
<img width="275" alt="Screen Shot 2020-04-14 at 17 14 13" src="https://user-images.githubusercontent.com/4999794/79279357-7ce5df80-7e73-11ea-9c83-dd23e0bb3838.png">

**Comment by htmgarcia** [comment]
@andergmartins authors box (and other such as metadata and notifications) uses different structure in comparison with the boxes from the WP core. This seems to make keyboard accessibility a bit complicated. Please see the screenshot from my comment above.

**Comment by andergmartins** [comment]
@htmgarcia you're right. They are displayed that way because we still rely on the "old" meta boxes. We don't provide those features as Gutenberg plugins for example. We keep using the compatibility layer with the legacy metaboxes. But I see your point, we need consistency on that too. I hope we are able to improve this soon. We have other accessibility issues like #63

**Comment by htmgarcia** [comment]
Not sure if something changed, but while testing today is possible to access the Author box and select at least one author through the keyboard.
